studio/src/editors/merch-card-editor.js

```diff
@@ -51,6 +51,14 @@ class MerchCardEditor extends LitElement {
                 value="${form.mnemonicIcon.values.join(',')}"
                 @change="${this.updateFragment}"
             ></sp-textfield>
+            <sp-field-label for="card-icon">Badge</sp-field-label>
+            <sp-textfield
+                placeholder="Enter yellow badge text"
```

Comment:
I don't like the "Yellow badge" thing.
It is yellow in the card design but in the form we should just call it badge text.
With the live preview it will look yellow anyway.

Reply:
right, will remove 'yellow' part

Reply:
and soon enough we might have badge text colour and background color anyways